src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplDownloadSteps.java

```diff
@@ -1,27 +1,55 @@
 package gov.healthit.chpl.aqa.stepDefinitions;
 
-import cucumber.api.java.en.Given;
-import cucumber.api.java.en.Then;
-import cucumber.api.java.en.When;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+
+import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang3.StringUtils;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.remote.CapabilityType;
+import org.openqa.selenium.remote.DesiredCapabilities;
 import org.openqa.selenium.support.ui.Select;
-import org.openqa.selenium.support.ui.WebDriverWait;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
 
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
 import gov.healthit.chpl.aqa.pageObjects.ChplDownloadPage;
-
 /**
  * Class ChplDownloadSteps definition.
  */
 public class ChplDownloadSteps {
 
     private WebDriver driver;
-    private static final int TIMEOUT = 30;
     private String url = System.getProperty("url");
+    private static String downloadPath = "\\filepath\\tempdnld";
```

Comment:
For this to work on linux, we'll need to have the download path set up as a property value, the same way the url is